The Shipyard.txt

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Since this funcitonality is being done in a different PR, we can probably remove this file from here.  It clutters things up and contributes little to the intended goal of this PR.

Plus its a text file that looks like it is trying to be a poorly formatted java file.

Reply:
I agree let's remove this file